bin/fin

```diff
@@ -198,7 +204,19 @@ CONFIG_DOCKER_MACHINE_ENV="$CONFIG_DOCKER_MACHINE_DIR/machine.env"
 CONFIG_CERTS=${CONFIG_CERTS:-$CONFIG_DIR/certs}
 
 # Host path to make accessible to the VM (Mac only)
-DOCKSAL_NFS_PATH="${DOCKSAL_NFS_PATH:-/Users}"
+# macOS Catalina required special treatment
+# See https://github.com/docksal/docksal/issues/1130
+if is_mac_catalina; then
+	if [[ "${DOCKSAL_NFS_PATH}" != "" ]] && ! [[ "${DOCKSAL_NFS_PATH}" =~ "/System/Volumes/Data" ]] && ! [[ "${DOCKSAL_NFS_PATH}" =~ "/Volumes" ]]; then
+		# Prefix user provided path with "/System/Volumes/Data" unless the prefix is already there AND the path is NOT on an external drive (/Volumes/...)
+		DOCKSAL_NFS_PATH="/System/Volumes/Data${DOCKSAL_NFS_PATH}"
+	else
+		# Either the default value or the value provided by user
+		DOCKSAL_NFS_PATH="${DOCKSAL_NFS_PATH:-/System/Volumes/Data/Users}"
+	fi
+else
+	DOCKSAL_NFS_PATH="${DOCKSAL_NFS_PATH:-/Users}"
+fi
```

Comment:
I don't think anyone will fix the thing that you cannot mount `/Users` via NFS anymore, it most likely will stay like this forever, because its so "by design". They would have to make `nfsd` very macos-specific to fix the issue.

So I guess users should be aware of what is happening, and in case they have custom NFS path already set, they should have to update it **manually** after upgrading to Catalina if needed.

With all that in mind, I would change it to this:

```suggestion
if is_mac_catalina; then
	DOCKSAL_NFS_PATH="${DOCKSAL_NFS_PATH:-/System/Volumes/Data/Users}"
else
	DOCKSAL_NFS_PATH="${DOCKSAL_NFS_PATH:-/Users}"
fi
```